src/components/tabset/tabset.component.html

```diff
@@ -0,0 +1,45 @@
+<!-- Nav tabs -->
+<ul role="tablist"
+    class="nav nav-{{ type }}" 
+    [class.nav-justified]="justified"
+    [class.nav-stacked]="vertical"
+    [class.minimal-tab]="minimal"
+    [attr.aria-label]="ariaLabel">
+
+	<li role="presentation" 
```

Comment:
I think if you set this to `role="none"` then it will remove the "list item" role implied by the li element. Current this is dictated as "1 of 4 list item."